features/feedback_link_summary.feature

```diff
@@ -0,0 +1,23 @@
+@US319
+Feature: Feedback prompt for users
+
+As a service provider
+I want to prompt users to provide feedback once they've completed their journey
+So that I can collect customer satisfaction scores and the service can be continually improved from feedback
+
+------------
+
+Text
+"What did you think of this service?
+
+Please give us your feedback (takes 30 seconds).
```

Comment:
I am not sure about '30 seconds'.

How about "It will not take much of your time" or the like?